molecule/default/tests/test_default.py

```diff
@@ -78,3 +78,54 @@ def test_open_ports(host):
     elif distribution == 'centos':
         assert host.socket("tcp://:::1515").is_listening
         assert not host.socket("tcp://:::1514").is_listening
+
```

Comment:
Hi @rshad,

I think we should move this to another issue and PR in order to keep track of the changes to the tests.

This way we can iterate and add new tests and keep a better overall view of the changes

Thanks!